app/views/catalog/_index_list_type_icon.html.erb

```diff
@@ -0,0 +1,19 @@
+<% case type.downcase %>
```

Comment:
I have a proposition about this. I think we can simplify this view. I wonder if we can add a method to the document.
`<%= render 'index_list_type_icon', type: document.type_label.first %>` could be changed to `<%= render 'index_list_type_icon', type: document.type_to_class %>` where #type_to_class (name pending) could translate the type to its font awesome class. Then this view could just have `<i class="fa #{fa_class}" aria-hidden="true"></i>` in it. I just feel like having a huge switch case in the view is messy.

Reply:
For sure. It was down to weighing a switch on a model that has almost no knowledge about FA and minimum custom methods or this view. Both messy in their own right.
I guess a hash could accomplish the same thing.

Check out 733b2cf and see if that's the direction you're thinking